npm-shrinkwrap.json

```diff
@@ -1363,7 +1573,7 @@
       "version": "1.1.7"
     },
     "graceful-fs": {
-      "version": "4.1.4"
+      "version": "1.2.3"
```

Comment:
Hmm, not sure why npm picked that version over 4.1.4. Maybe it tries to choose the lowest version as root? https://docs.npmjs.com/how-npm-works/npm3-dupe

```
>$ npm ls graceful-fs
wp-calypso@0.17.0 /Users/kerryliu/checkouts/wp-calypso
├─┬ eslint@1.10.3
│ └─┬ file-entry-cache@1.2.4
│   └─┬ flat-cache@1.0.10
│     ├── graceful-fs@4.1.4 
│     └─┬ read-json-sync@1.1.1
│       └── graceful-fs@4.1.4 
├─┬ mocha@2.3.4
│ └─┬ glob@3.2.3
│   └── graceful-fs@2.0.3 
├─┬ node-sass@3.4.2
│ ├─┬ gaze@0.5.2
│ │ └─┬ globule@0.1.0
│ │   └─┬ glob@3.1.21
│ │     └── graceful-fs@1.2.3 
│ ├─┬ meow@3.7.0
│ │ └─┬ read-pkg-up@1.0.1
│ │   └─┬ read-pkg@1.1.0
│ │     ├─┬ load-json-file@1.1.0
│ │     │ └── graceful-fs@4.1.4 
│ │     └─┬ path-type@1.1.0
│ │       └── graceful-fs@4.1.4 
│ └─┬ node-gyp@3.3.1
│   ├─┬ fstream@1.0.9
│   │ └── graceful-fs@4.1.4 
│   └── graceful-fs@4.1.4 
├─┬ postcss-cli@2.5.1
│ └─┬ chokidar@1.5.1
│   ├─┬ fsevents@1.0.12
│   │ └─┬ node-pre-gyp@0.6.25
│   │   └─┬ tar@2.2.1
│   │     └─┬ fstream@1.0.8
│   │       └── graceful-fs@4.1.3 
│   └─┬ readdirp@2.0.0
│     └── graceful-fs@4.1.4 
├─┬ webpack@1.12.15
│ ├─┬ enhanced-resolve@0.9.1
│ │ └── graceful-fs@4.1.4 
│ └─┬ watchpack@0.2.9
│   └── graceful-fs@4.1.4 
└─┬ wpcom@4.9.13
  └─┬ babel@5.8.38
    └─┬ babel-core@5.8.38
      └─┬ regenerator@0.8.40
        └─┬ commoner@0.10.4
          └── graceful-fs@4.1.4 
```